trezorlib/client.py

```diff
@@ -254,7 +254,7 @@ def callback_RecoveryMatrix(self, msg):
                 return proto.WordAck(word='\x08')
 
             # ignore middle column if only 6 keys requested.
-            if (isinstance(msg.type, types.WordRequestType_Matrix6) and character in ('2', '5', '8')):
+            if msg.type == types.PinMatrixRequestType_Current and character in ('2', '5', '8'):
```

Comment:
Is change from WordRequestType_Matrix6 to WordRequestType_Matrix6 intended or accidental?